static/js/components/GcnNoticeTypesSelect.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import PropTypes from "prop-types";
+
+import SelectWithChips from "./SelectWithChips";
+import { fetchGcnNoticeTypes } from "../ducks/gcnNoticeTypes";
+
+const GcnNoticeTypesSelect = (props) => {
+  const { selectedGcnNoticeTypes, setSelectedGcnNoticeTypes } = props;
+  const dispatch = useDispatch();
+  const gcn_notice_types = useSelector(
+    (state) => state.gcnNoticeTypes.gcnNoticeTypes
+  );
+
+  useEffect(() => {
+    dispatch(fetchGcnNoticeTypes());
+  }, []);
```

Comment:
If you add this to the `config_handler` it should be loaded into the state when the app is "hydrated".

Reply:
Sounds good !